src/services/formatting/smartIndenter.ts

```diff
@@ -464,9 +471,29 @@ namespace ts.formatting {
                 case SyntaxKind.GetAccessor:
                 case SyntaxKind.SetAccessor:
                     return child !== SyntaxKind.Block;
+                case SyntaxKind.ExportDeclaration:
+                    return child !== SyntaxKind.NamedExports;
                 default:
                     return false;
             }
         }
+
+        function hasBraceShell(node: Node) {

```

Comment:
@vladima I was wrong and the braces are not optional. I just removed `hasBraceShell` and added a simpler check.